lab/manifests/system/base/cilium/internal/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+helmCharts:
+- name: cilium
+  valuesInline:
```

Comment:
Nit: maybe move this field down to the last position to not confuse "here are helm values, and here we are back to helm chart value"